Gems/ROS2/Code/Source/ContactSensor/ROS2ContactSensorComponent.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2ContactSensorComponent.h"
+#include "ArticulationLinkComponent.h"
```

Comment:
I don't think this header is used. Please review the include list to see if there are any other unused ones.

Reply:
Fixed